brokerapi/brokers/bigtable/broker.go

```diff
@@ -139,7 +141,7 @@ func (b *BigTableBroker) Deprovision(instanceID string, details models.Deprovisi
 
 	instance := models.ServiceInstanceDetails{}
 	if err = db_service.DbConnection.Where("ID = ?", instanceID).First(&instance).Error; err != nil {
-		return models.ErrInstanceDoesNotExist
+		return brokerapi.ErrInstanceDoesNotExist
```

Comment:
Same question as previous about callers doing a type assertion on the old error type...